src/main/java/com/feywild/feywild/block/trees/FeyPlankBlock.java

```diff
@@ -0,0 +1,58 @@
+package com.feywild.feywild.block.trees;
+
+import io.github.noeppi_noeppi.libx.base.decoration.DecoratedBlock;
+import io.github.noeppi_noeppi.libx.base.decoration.DecorationContext;
+import io.github.noeppi_noeppi.libx.base.decoration.DecorationType;
+import io.github.noeppi_noeppi.libx.mod.ModX;
+import net.minecraft.core.Direction;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.context.BlockPlaceContext;
+import net.minecraft.world.level.block.*;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.BlockStateProperties;
+
+import javax.annotation.Nonnull;
+
+public class FeyPlankBlock extends DecoratedBlock {
+
+    private static final DecorationContext DECORATION = new DecorationContext("fey_plank", DecorationType.BASE, DecorationType.FENCE, DecorationType.FENCE_GATE, DecorationType.SLAB, DecorationType.STAIR);
```

Comment:
Is there a reason for this being private? I think the information, a `DecorationType` offers may be useful for stuff using the plank blocks, so I think making it public would be better.